setup.py

```diff
@@ -34,17 +33,26 @@
         "Topic :: Software Development :: Libraries :: Python Modules",
         "Topic :: Software Development :: Quality Assurance",
     ],
+    project_urls={
+        'Issue tracker': 'https://github.com/python-discord/flake8-annotations/issues',
+        'Discord server': 'https://discord.gg/python',
+    },
     install_requires=[
```

Comment:
Need `typed-ast` in install_requires, otherwise someone installing via pip (versus pipenv) will not get it.